examples/nextjs/pages/_app.js

```diff
@@ -0,0 +1,15 @@
+import App from 'next/app'
+import React from 'react'
+import ThemeProvider from '@datapunt/asc-ui/lib/theme/ThemeProvider'
```

Comment:
Should we not be importing this from the index of `@datapunt/asc-ui` instead?

Reply:
Yes, we can do that now!